### DISCUSSION TITLE
Selecting the surface elements for a cylindrical geometry for ABAQUS input

### DISCUSSION THREAD

**Discussion by skunda-uw** [neperfepx/neper#737] [Q&A] [answered]
I want to create surfaces out of elements for an ABAQUS analysis. The problem domain is cylindrical with 200 tesellations. It has been meshed using quadratic tetrahedrons. I want to create 3 surfaces : one each for the two flat, circular end faces of the cylinder and a third for the curved surface of the cylinder. I thought the following two lines of code would be enough to accomplish this
```
neper -T -n 200 -reg 1 -id 2 -domain "cylinder(6, 6)"
```
```
neper -M n200-id2.tess -order 2 -o newmesh -format inp -rcl 0.8 -faset "faces"
```
It gives me only the node sets for the three surfaces described above. Not the corresponding elements. In fact, only the element sets corresponding to each of the tesellations are created by these two commands. How do I get Neper to output the element sets corresponding to the surfaces of the cylinder ?

Ultimately, I want to name the element sets `TopSurfaceElements`, `BaseSurfaceElements` and `FreeSurfaceElements` and have the following three lines in my ABAQUS input file for the creation of the surfaces

```
*Surface, name=TopSurface, type=element                                                                                                                                                        
TopSurfaceElements,S2
*Surface, name=BaseSurface, type=element
BaseSurfaceElements,S1
*Surface, name=FreeSurface, type=element
FreeSurfaceElements,S3
```
I'm not sure how to correctly specify the face of the elements for the surface creation.

**Comment by rquey** [comment, marked as answer]
Please give a try to the latest version (`4.6.1-9`) and let me know if it works.  One thing is that you will get several `*Surface`s for the "free" surface.
